templates/thank-you.html

```diff
@@ -1,94 +1,105 @@
-{% extends 'layouts/_base.html' %}
+{% extends "layouts/_base.html" %}
+
 {% from 'components/panel/_macro.njk' import onsPanel %}
 {% from "components/description-list/_macro.njk" import onsDescriptionList %}
 {% from "components/timeout-panel/_macro.njk" import onsTimeoutPanel %}
 
 {% set page_title = _("We’ve received your answers") %}
 {% set hide_sign_out_button = content.hide_sign_out_button %}
 {% if account_service_todo_url %}
-  {% set breadcrumbs = {
-    "ariaLabel": 'Back to surveys',
-    "itemsList": [
-      {
-        "url": account_service_todo_url,
-        "id": "back-to-surveys",
-        "text": _("Back to surveys"),
-        "attributes": {
-          "data-ga": 'click',
-          "data-ga-category": 'Navigation',
-          "data-ga-action": 'Back to surveys link click'
-        }
-      }
-    ]
-  } %}
+    {% set breadcrumbs = {
+            "ariaLabel": 'Back to surveys',
+            "itemsList": [
+                {
+                    "url": account_service_todo_url,
+                    "id": "back-to-surveys",
+                    "text": _("Back to surveys"),
+                    "attributes": {
+                        "data-ga": 'click',
+                        "data-ga-category": 'Navigation',
+                        "data-ga-action": 'Back to surveys link click'
+                    }
+                }
+            ]
+        } %}
```

Comment:
Think the formatting is a bit wrong here, looks like its been indented a level too far